dev/ProgressBar/ProgressBar.idl

```diff
@@ -27,6 +27,19 @@ unsealed runtimeclass ProgressBarTemplateSettings : Windows.UI.Xaml.DependencyOb
 
     Windows.UI.Xaml.Media.RectangleGeometry ClipRect;
     static Windows.UI.Xaml.DependencyProperty ClipRectProperty{ get; };
+
+    // TemplateSetting properties from WUXC for backwards compatibility.
```

Comment:
Please add tests for ProgressBar and ProgressRing where the template is set to the old template from WUXC. And another with the template for ProgressRing with a custom lottie animation source.